app/test/core/services/database/test_data.dart

```diff
@@ -10,6 +10,14 @@ Map<String, dynamic> fakeData() {
   return _fakeResponse;
 }
 
+Map<String, dynamic> fakeInitialAtr() {
```

Comment:
- [ ] Please move `test_data`, `test_helper` with the mocks into the `lib/test/` folder.

I was helping Mansi with `AuthenticationService` tests, which uses these files. Without being under `lib` the files have relative imports which are annoying.